klock/src/jvmTest/kotlin/com/soywiz/klock/KlockLocaleTestJvm.kt

```diff
@@ -0,0 +1,181 @@
+package com.soywiz.klock
+
+import com.soywiz.klock.locale.*
+import org.junit.Ignore
+import org.junit.Test
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+import java.util.*
+import kotlin.test.assertEquals
+
+class KlockLocaleTestJvm {
+
+    private val outputPattern = "EEEE, d MMMM yyyy HH:mm:ss"
+
+    private val klockDate = DateTime(
+        year = 1995,
+        month = Month.January,
+        day = 18,
+        hour = 21,
+        minute = 36,
+        second = 45
+    )
+    private val javaDate = LocalDateTime.of(
+        1995, // year
+        java.time.Month.JANUARY, // month
+        18, // day of month
+        21, // hour
+        36, // minute
+        45 // second
+    )
+
+
+    @Test
+    fun assertDatesAreTheSame() {
+        val klockLong = klockDate.unixMillisLong
+        val javaLong = javaDate.toInstant(ZoneOffset.UTC).toEpochMilli()
+        assertEquals(javaLong, klockLong)
+    }
+
+    private fun LocalDateTime.format(pattern: String, locale: Locale) = this.format(
+        DateTimeFormatter.ofPattern(pattern, locale)
+    )
+
+    @Test
+    fun assertEnglishLocalization() {
+        val javaOutput = javaDate.format(outputPattern, Locale.ENGLISH)
+
+        val klockOutput = KlockLocale.setTemporarily(KlockLocale.english) {
+            DateFormat(outputPattern).format(klockDate)
+        }
+
+        assertEquals(javaOutput, klockOutput)
+    }
```

Comment:
This code is duplicated.
Can you create a utility function receiving a Locale + KlockLocale that makes the assert?

Reply:
Yes, thanks. Should be more readable and easier to add more tests now.